include/fast_float/ascii_number.h

```diff
@@ -13,6 +13,7 @@ namespace fast_float {
 // Next function can be micro-optimized, but compilers are entirely
 // able to optimize it well.
 fastfloat_really_inline bool is_integer(char c)  noexcept  { return c >= '0' && c <= '9'; }
+fastfloat_really_inline bool is_hexidecimal(char c)  noexcept { return c >= '0' && c <= '9' || (c >= 'a' && c <= 'f') || (c >= 'A' && c <= 'F'); }
```

Comment:
is_hexidecimal or is_hexadecimal

Reply:
Forgot that went unused, could remove that.